config/ember-try.js

```diff
@@ -1,11 +1,36 @@
 /*jshint node:true*/
 module.exports = {
-  useVersionCompatibility: true,
```

Comment:
We **do** lose some version test coverage here, but it is more inline with what ember-cli recommends so I'm OK with it. Do we need to change our `versionCompatibility` in package.json because of this (or remove it?)